tests/ruby/i18n_spec.rb

```diff
@@ -24,5 +24,78 @@ module Yast
         expect(Nn_(singular, plural, count)).to eq [singular, plural, count]
       end
     end
+
+    describe "._" do
+      TRANSLATED = "translated".freeze
+      SINGULAR = "untranslated".freeze
+      PLURAL = "plural".freeze
+
+      before do
+        allow(File).to receive(:exist?).with(Yast::I18n::LOCALE_DIR)
+          .and_return(true)
+        textdomain("base")
+      end
+
+      it "returns the translated string" do
+        allow(FastGettext).to receive(:key_exist?).and_return(true)
+        expect(FastGettext::Translation).to receive(:_).with(SINGULAR)
+          .and_return(TRANSLATED)
+        expect(_(SINGULAR)).to eq(TRANSLATED)
+      end
+
+      context "when FastGettext throws an exception" do
```

Comment:
well, in fact it is not "throws an exception" but one specific type of exception.